### PR TITLE
Feature/nex 1952/support tool jwks

### PR DESCRIPTION
**Pull request by siwane**
- Allow to cache item with numerical ttl in second.
- Set isHit to true if item is retrieved from cache

### CODE REVIEW

File: common/oatbox/cache/CacheItem.php [hunk 1]
```diff
@@ -120,6 +120,10 @@ public function expiresAfter($time): self
             return $this;
         }
 
+        if (is_int($time)) {
+            $time = new DateInterval(sprintf('PT%dS', $time));
```
Comment (mike-ionut-mihai-sandu-tao) [inline common/oatbox/cache/CacheItem.php:124]: DateInterval::format

Comment (siwane) [reply common/oatbox/cache/CacheItem.php:124]: I do not understand the question

**Review by gabrielfs7 — APPROVED** [review]
Code LGTM. Good job!

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**